### PR TITLE
[downward] Reduce build failure

### PR DESCRIPTION
**Pull request by furushchev**
This PR includes:
- use `ExternalProject` module instead of ros `mk` package
- add test case for simple planning of pddl

### CODE REVIEW

File: 3rdparty/downward/package.xml [hunk 1]
```diff
@@ -13,14 +13,10 @@
...
-  <build_depend>mercurial</build_depend>
   <build_depend>python</build_depend>
   <build_depend>flex</build_depend>
   <build_depend>bison</build_depend>
   <build_depend>gawk</build_depend>
   <build_depend>g++-static</build_depend>
-  <build_depend>rospack</build_depend>
-  <build_depend>roslib</build_depend>
-  <build_depend>mk</build_depend>
-
```
Comment (garaemon) [inline 3rdparty/downward/package.xml]: Need git

Comment (garaemon) [inline 3rdparty/downward/package.xml]: Oops, it does not need git.

File: 3rdparty/downward/CMakeLists.txt [hunk 2]
```diff
@@ -5,18 +5,21 @@ find_package(catkin REQUIRED)
 
 catkin_package()
 
-add_custom_command(
-  OUTPUT installed
-  COMMAND \$\(MAKE\) -f ${PROJECT_SOURCE_DIR}/Makefile
```
Comment (garaemon) [inline 3rdparty/downward/CMakeLists.txt]: see https://github.com/jsk-ros-pkg/jsk_3rdparty/blob/master/3rdparty/libcmt/CMakeLists.txt#L9

**Comment by furushchev** [comment]
rebased. it should pass test.

**Comment by garaemon** [comment]
See https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/35#discussion-diff-41220066

File: 3rdparty/downward/CMakeLists.txt (hunk 2 above)
Comment (furushchev) [inline 3rdparty/downward/CMakeLists.txt]: You mean, we should add following lines?

```cmake
execute_process(COMMAND
  git config --global http.sslVerify false)
```

I use tarball, so I think we dont need to add this code?

Comment (garaemon) [inline 3rdparty/downward/CMakeLists.txt]: I see, it is not git. You don't need to add those lines.
Please check [ros build farm](http://www.ros.org/debbuild/indigo.html) after merge it **carefully**.

Comment (furushchev) [inline 3rdparty/downward/CMakeLists.txt]: I'm very sorry, but I still cannot understand.

```diff
-add_custom_command(
-  OUTPUT installed
-  COMMAND \$\(MAKE\) -f ${PROJECT_SOURCE_DIR}/Makefile
```

I didn't add but remove these lines.
What is better solution which I should do?

Comment (garaemon) [inline 3rdparty/downward/CMakeLists.txt]: > I didn't add but removed these lines.

It's ok. Remove them.

**Comment by furushchev** [comment]
@garaemon Thanks for reviewing & merging :+1:

**Comment by garaemon** [comment]
[2.0.10](https://github.com/jsk-ros-pkg/jsk_3rdparty/releases/tag/2.0.10) has been released.

**Pull request merged**